CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 0.1.2
+
+- Handle superfluous `version` keys with `git` and `path` dependencies.
```

Comment:
[nit] Maybe change "Handle" to "Allow" since we don't do anything with them?

Reply:
ack